test/TestGrains/RequestContextTestGrain.cs

```diff
@@ -40,6 +40,7 @@ public Task<Guid> E2EActivityId()
 
         public Task<Guid> E2ELegacyActivityId()
         {
+            if (!RequestContext.PropagateActivityId) throw new OrleansException("ActivityId Propegation is not enabled on silo.");
```

Comment:
typo in `Propegation`

Reply:
BTW, this should be InvalidOperationException or some other non-Orleans runtime exception, right?